final_prep/README.md

```diff
@@ -19,37 +19,37 @@ Some things to consider as you refactor include...
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![Screen Shot 2021-08-07 at 10 24 57 AM](https://user-images.githubusercontent.com/81737385/128607045-e91bbb43-80e0-4e22-9e70-883c7ac24f50.png)
+- ![Screen Shot 2021-08-07 at 10 25 32 AM](https://user-images.githubusercontent.com/81737385/128607050-a1b787da-9b0e-4a14-b81b-14eb20528a86.png)
+- ![Screen Shot 2021-08-07 at 10 25 43 AM](https://user-images.githubusercontent.com/81737385/128607060-cea0991c-0675-43d9-891f-cedfafe4b509.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/JCNapier/35a726bc1c96c2ad4aab9a96b49f6435
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: https://gist.github.com/JCNapier/81f3c3c2f31b29345cf75c646f9db727
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: https://gist.github.com/JCNapier/f1678dc15f1deebdd16cab385cd789fd
```

Comment:
Nice self awareness here. During Turing, I had a situation where my mom had to have unexpected surgery on the day of an evaluation. This was not a fun day. Things were fine for my mom, but there was uncertainty for some time there, and it was important that I not keep my feelings bottled up, and express some of that to classmates, and my instructors. Sometimes, the tendency can be to try to put your head down and plow forward with work, but development is such focused work that your head can get out of it, and before you know it, 2 hours just went down the drain when you could've at least been working to process what is going down in life- at that time, I needed a release before I could focus again. Wishing well for you and your family as this journey starts!